.github/ISSUE_TEMPLATE/---category-suggestion.yml

```
@@ -0,0 +1,52 @@
+---
+name: "Category Request ⚡"
+description: "I would like to request a feature."
```

```suggestion
description: "Use this to add new category/subcategory into the Hub"
```

This description aligns with the format used in other issue forms within the repository, ensuring consistency.